### ISSUE TITLE
worth mentioning pre-built Linux packages in README

### ISSUE THREAD

**Issue by SingingBush** [flaviotordini/musique#19]
I'm a Fedora user (v29) and on finding the project scanned the readme, cloned the code and ran:

```
$ sudo dnf install -y qt5-devel taglib-devel qtav-devel
$ qmake
$ make
```

It was only after building that I found there's already prebuilt packages available on https://pkgs.org/download/musique

It's worth putting that into the readme.

nice work btw, this music player is just what I want. My only gripe is that I prefer to see album art that's attached to the file rather than attempting to find it online.

**Comment by flaviotordini** [comment]
A bit out of scope, IMO. Musique does extract the album from files before getting them from Last.fm.
